x-pack/plugins/cases/public/components/case_view/case_view_metrics.tsx

```diff
@@ -23,34 +23,38 @@ const MetricValue = styled(EuiFlexItem)`
 
 export interface CaseViewMetricsProps {
   metrics: CaseMetrics | null;
+  features: CaseMetricsFeature[];
   isLoading: boolean;
 }
 
 type MetricItems = Array<{ title: string; value: number }>;
 
-const useMetricItems = (metrics: CaseMetrics | null): MetricItems => {
+const useMetricItems = (
+  metrics: CaseMetrics | null,
+  features: CaseMetricsFeature[]
+): MetricItems => {
   const { alerts, connectors } = metrics ?? {};
-  const totalConnectors = connectors?.length;
-  const alertsCount = alerts?.count;
-  const totalAlertUsers = alerts?.users?.total;
-  const totalAlertHosts = alerts?.hosts?.total;
+  const totalConnectors = connectors?.length ?? 0;
+  const alertsCount = alerts?.count ?? 0;
+  const totalAlertUsers = alerts?.users?.total ?? 0;
+  const totalAlertHosts = alerts?.hosts?.total ?? 0;
 
   const metricItems = useMemo<MetricItems>(() => {
     const items = [];
-    if (alertsCount != null) {
+    if (features.includes('alerts.count')) {
```

Comment:
super nit: You could alternatively loop over `features` (or reduce) and do a switch case instead of `includes`.

Reply:
I did it that way to show the KPIs always in the same order, if I loop the `features` array the order will depend on the `features = { metrics: [...] }` property that each solution pass. Do you think that's fine?